server/models/services/AtService.js

```diff
@@ -1,13 +1,11 @@
 const ModelService = require('./ModelService');
-
 const {
     AT_ATTRIBUTES,
     AT_VERSION_ATTRIBUTES,
     AT_MODE_ATTRIBUTES,
     BROWSER_ATTRIBUTES
 } = require('./helpers');
 const { Sequelize, At, AtVersion, AtMode } = require('../');
-const { clearCachedAts } = require('../loaders/utils');
```

Comment:
Can you remind me of the caching issue and the reasoning behind removing this?

Reply:
Hopefully my big explanatory comment will clear this up! https://github.com/w3c/aria-at-app/pull/951#discussion_r1529061043